packages/boot/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@agoric/boot",
+  "version": "0.1.0",
+  "description": "Config and utilities to bootstrap an Agoric chain",
+  "type": "module",
+  "main": "./index.js",
+  "repository": "https://github.com/Agoric/agoric-sdk",
+  "scripts": {
+    "build": "exit 0",
+    "prepack": "tsc --build jsconfig.build.json",
+    "postpack": "git clean -f '*.d.ts*'",
+    "test": "ava",
+    "test:xs": "SWINGSET_WORKER_TYPE=xs-worker ava 'test/bootstrapTests/**/test-*.js' 'test/upgrading/**/test-*.js'",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint:eslint": "eslint ."
+  },
+  "keywords": [],
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/ertp": "^0.16.2",
+    "@agoric/internal": "^0.3.2",
+    "@agoric/vat-data": "^0.5.2",
+    "@agoric/vats": "^0.15.1",
+    "@agoric/zoe": "^0.26.2",
+    "@endo/marshal": "^0.8.6",
+    "@endo/bundle-source": "^2.5.2",
+    "@endo/captp": "^3.1.2",
+    "@endo/eventual-send": "^0.17.3",
+    "@endo/init": "^0.5.57",
+    "@endo/far": "^0.2.19",
+    "@endo/promise-kit": "^0.2.57",
+    "@endo/stream": "^0.3.26",
+    "import-meta-resolve": "^2.2.1"
+  },
+  "devDependencies": {
+    "@agoric/assert": "^0.6.0",
+    "@agoric/cosmic-swingset": "^0.41.3",
+    "@agoric/deploy-script-support": "^0.10.3",
+    "@agoric/governance": "^0.10.3",
+    "@agoric/inter-protocol": "^0.16.1",
+    "@agoric/store": "^0.9.2",
+    "@agoric/swing-store": "^0.9.1",
+    "@agoric/swingset-liveslots": "^0.10.2",
+    "@agoric/swingset-vat": "^0.32.2",
+    "@agoric/time": "^0.3.2",
+    "ava": "^5.3.0",
+    "c8": "^7.13.0"
+  },
+  "files": [
+    "CHANGELOG.md",
+    "src/",
```

Comment:
In some ways it would be more clear to also move `lib-boot.js`, `basic-behaviors.js`, and the other stuff that only happens at bootstrap.

I gather that would not help with untying the dependency knots?